frontend/src/components/MainSearch/MainSearch.js

```diff
@@ -16,6 +17,16 @@ class MainSearch extends Component {
     this.togglePanel = this.togglePanel.bind(this);
   }
 
+  componentDidMount() {
+    const cookies = new Cookies();
+    if (cookies.get('visit') !== 'set') {
+      this.setState({
+        open: true,
```

Comment:
NitPick: I think we could be a little more descriptive than just `open`, since that could refer to anything, not just the search instructions. Let's do something like `showSearchInstructions`.